docs/recipes/ec2-eks-metrics-go-adot-ampamg.md

```diff
@@ -1,12 +1,314 @@
 
 ## Infrastructure
 
+### Architecture
+
+The ADOT-AMP pipeline enables us to use the ADOT Collector to scrape a Prometheus-instrumented application, and send the scraped metrics to AMP. 
+
+![Architecture](https://aws-otel.github.io/static/Prometheus_Pipeline-07344e5466b05299cff41d09a603e479.png)
+
+The ADOT-AMP pipeline includes two OpenTelemetry Collector components specific to Prometheus — the Prometheus Receiver and the AWS Prometheus Remote Write Exporter. 
+
+[Getting Started with Prometheus Remote Write Exporter for AMP](https://aws-otel.github.io/docs/getting-started/prometheus-remote-write-exporter)
+
+
+### Prerequisites
+
+* The AWS CLI is [installed](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) and [configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) in your environment.
+* You need to install the [eksctl](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html) command in your environment.
+* You need to install [kubectl](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html) in your environment. 
+
+### Setup an EKS cluster
+
+For this recipe we require a EKS cluster to be available.
+You can either use an existing EKS cluster or create one using [cluster_config.yaml](ec2-eks-metrics-go-adot-ampamg/cluster_config.yaml).
+
+This template will create a new cluster with [AWS Fargate](https://aws.amazon.com/fargate/) enabled. 
+
+Edit the template file and set your region to one of the available regions for AMP:
+
+* `us-east-1`
+* `us-east-2`
+* `us-west-2`
+* `eu-central-1`
+* `eu-west-1`
+
+Make sure to overwrite this region in your bash session for example:
+```
+export AWS_DEFAULT_REGION=eu-west-1
+```
+Other regions are currently unsupported for AMP.
+
+Create your cluster using the following command.
+```
+eksctl create cluster -f cluster-config.yaml
+```
+
+### Setup an ECR repository
+
+Create an ECR repository for the sample application.
+
+```
+aws ecr create-repository \
+    --repository-name prometheus-sample-app \
+    --image-scanning-configuration scanOnPush=true \
+    --region eu-west-1
+```
+
+### Setup AMP 
+
+#### Create AMP workspace
+
+create a workspace using the AWS CLI 
+```
+aws amp create-workspace --alias prometheus-sample-app
+```
+Take note of the returned workspace-id
+
+Verify the workspace is created using:
+```
+aws amp list-workspaces
+```
+For more details check out the [AMP Getting started](https://docs.aws.amazon.com/prometheus/latest/userguide/AMP-getting-started.html) guide.
+
+#### Setup IAM role permissions for scraping
+
+1. Create the IAM role for the service account by following the steps in [Set up service roles for the ingestion of metrics from Amazon EKS clusters](https://docs.aws.amazon.com/prometheus/latest/userguide/set-up-irsa1.html#set-up-irsa-ingest). The AWS Distro for Open Telemetry Collector will use this role when it scrapes and exports metrics
```

Comment:
It is not concern to this PR, but the doc https://docs.aws.amazon.com/prometheus/latest/userguide/set-up-irsa.html#set-up-irsa-ingest says to replace my_prometheus_namespace, it is is not very clear if workspace is same as namespace, if not where do I find the namespace. If same, may be add notes to above capture workspace id section, this might also be referred as namespace in IRSA sections.